### PR TITLE
Add Start with python

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
and typo SLASH

### CODE REVIEW

**Comment by moritzpossecker** [comment]
Thanks for your suggestions Markus! 

I'd like to implement some of them, but not all align perfectly with my current needs for this repo. As the API is solely for data retrieval, a user interface isn't necessary.

I apprecciate you correcting the README typo. How about creating a specific pull request just for that? 
I also like the approach of calling the endpoints using `/second_page?url=...`

Additionally, I could use some help right now in:
- breaking down the `fussballdescraper.py` into smaller, more manageable modules
- developing unit tests for both endpoints

Either way - thanks for contributing and thanks for your input!

Kind regards, Moritz

**Comment by MarkusHackspacher** [comment]
Thank for your answer Mortiz,

yes you seen i make a UI to use your Code, I did this because I had problems entering into the API Names with umlauts (ß,ä,ü,ö).
To use the UI, start at 127.0.0.1:5000 copy the league_url in the input field, second page select the team, third page, see the result.

And your right you need not a UI, but the UI is an easy way if others want to use your code, and it doesn't break your api.

Maybe it is possible to include these in separate files marked gui_....py in your project. Or second_gui

For additionally help, best way is open a issues. Is better to manage.
https://github.com/moritzpossecker/lockertunes-fussballde-api/issues

Thank you, Markus

**Comment by moritzpossecker** [comment]
Yeah ok, I see... 
A console application may be enough for that reason, but I understand and definitely would agree with you there about implementing something like that in some way. 
Like I said my priority would be to implement some unit tests.

**Comment by MarkusHackspacher** [comment]
About that 
> implementing something like that in some way

Have you started my code and what do you think of the concept?

**Comment by moritzpossecker** [comment]
Yes I have started it and I think the concept is good, I'd recommend a page for each endpoint and always a link to the actual API request result.

File: main.py [hunk 1]
```diff
@@ -34,5 +37,38 @@ def request_matches(team_name: str, league_url: str, game_span: int):
...
+        return redirect(url_for('second_page',  
+ url=url))
+    return render_template('index.html')
+
+@app.route('/second_page', methods=['GET', 'POST'])
+def second_page():
+    url = request.args.get('url')
+    teams = get_teams(url)
+    if request.method == 'POST':
+        selected_team = request.form['team']
```
Comment (MarkusHackspacher) [inline main.py:54]: The call teams = get_teams(url) is displayed directly on the page

File: main.py [hunk 2]
```diff
@@ -34,5 +37,38 @@ def request_matches(team_name: str, league_url: str, game_span: int):
...
+def third_page():
+    url = request.args.get('url')
+    selected_team = request.args.get('selected_team')
+    if selected_team:
+        matches = get_matches(selected_team, url, 50)
+        matches_dict = []
+        for match in matches:
+            matches_dict.append(match.to_dict())
+        if request.method == 'POST': 
+            return jsonify(matches_dict), 202
```
Comment (MarkusHackspacher) [inline main.py:69]: is the same as in the API function, this can certainly be solved by calling the function directly.

**Pull request closed without merging**